Docs/Configuration.tex

```diff
@@ -1298,8 +1296,8 @@ \subsection{MmioWhitelist Properties}\label{booterpropsmmio}
 \item
   \texttt{Enabled}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
-  \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: This address will be devirtualised unless set to \texttt{true}.
+  \textbf{Failsafe}: \texttt{false} (Devirtualise address)\\
+  \textbf{Description}: Virtualise address.
```

Comment:
This explanation is very bad as it does not explain that this whole list is rather a list of exclusions. Perhaps `Failsafe` should be `(Process as normal)` and the description should be `Exclude MMIO address from the devirtualisation procedure.` or something close.

Reply:
Just removed failsafe explanation since it can be expected to be the opposite of 'true'

Reply:
Well, just `Virtualise address` may be really unclear.

Reply:
Sorry, meant I used your suggested description and removed the failsafe one